### ISSUE TITLE
First Track of Session Does Not Record

### ISSUE THREAD

**Issue by riegelstamm** [jwallet/spy-spotify#194]
**What is the bug ?**
> The first track of a new session does not record.

**How to reproduce it or what's the expected behavior ?**
1. Using VB-Audio Virtual Cable, Minimal Length 30, MP3, LastFM or Spotify API (doesn't matter), Mute Ads On
2. Start playing in Spotify (does not matter which track)
3. Spytify says "Spying" but not "Recording"
4. Stop the track in Spotify and rewind to the beginning
5. Play the track in Spotify again
6. Spytify changes to "Recording"

**Screenshots**
![Capture](https://user-images.githubusercontent.com/10802064/76163639-cbd28380-6115-11ea-8c85-4656b2dc75fc.PNG)

**Comment by jwallet** [comment]
make sure to start Spytify before starting Spotify, it's actually the right behavior, don't want to start recording a song half way right?

**Comment by riegelstamm** [comment]
That's not the problem. Of course I start Spytify before playing a track, album, or playlist in Spotify. The second and following tracks record just fine; the first track does not record at all.

**Comment by jwallet** [comment]
spytify will start recording if Spotify call the event NextTrack or the event PlayPause changed, like this

![like](https://user-images.githubusercontent.com/23088305/76489919-1f87ea00-6400-11ea-88b8-9beddf31b690.gif)

**Comment by riegelstamm** [comment]
I made a video of the steps. As you can see, Spotify wasn't even running yet when I clicked Start in Spytify. However, the same thing happens if Spotify is already open (but not playing yet). My steps are these:

1. Start Spying (I left out this step in my list before).
2. Start playing track in Spotify
3. Spytify says "Spying" but not "Recording"
4. Stop the track in Spotify and rewind to the beginning
5. Play the track in Spotify again
6. Spytify changes to "Recording"

The video is here: https://drive.google.com/open?id=16JNBtCuZ-YcbXtFbggyLqAQDN3CW7Lyo
I hope you can see what is going on here.

**Comment by jwallet** [comment]
thanks for your feedback and the video, it was really helpful, actually help to find another issue
